docs/dev/table/streaming/temporal_tables.md

```diff
@@ -20,103 +20,255 @@ software distributed under the License is distributed on an
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
--->
+--> 
 
-Temporal Tables represent a concept of a (parameterized) view on a changing table that returns the content of a table at a specific point in time.
+Temporal table is a table that evolves over time as known as Flink [dynamic table](dynamic_tables.html), rows in temporal table are associated with one or more temporal periods, all Flink tables are temporal(dynamic) table.
```

Comment:
```suggestion
A temporal table is a table that evolves over time -  otherwise known in Flink as a [dynamic table](dynamic_tables.html). Rows in a temporal table are associated with one or more temporal periods and all Flink tables are temporal(dynamic).
```